src/api/ledger/pathfind.js

```diff
@@ -46,8 +46,12 @@ function requestPathFind(remote, pathfind: PathFind, callback) {
       _.omit(utils.common.toRippledAmount(amount), 'value'));
   }
 
-  remote.requestRipplePathFind(params,
-    composeAsync(_.partial(addParams, params), callback));
+  remote.createPathFind(params, function(data) {
+    if (data.full_reply) {
```

Comment:
@wltsmrz  raises a good point I think. What's the error handling situation here?